docs/Command-List.md

```diff
@@ -542,3 +546,19 @@ Options:
   -P, --platform TEXT  Specify the platform  [default: amd64]
   -h, --help           Show this message and exit.
 ```
+
+
+## iotedgedev solution tag
+
+```text
+Usage: iotedgedev solution tag [OPTIONS]
+
+  Adds tags to device twin for layered deployments
```

Comment:
>layered [](http://example.com/codeflow?start=31&length=7)

should it say automated deployments instead?

Reply:
```suggestion
  Adds tags to device twin for automatic deployments
```

Reply:
I've added additional changes for this PR here: #507 
(I'm unable to change new files in this PR due to lack of permissions on Tamaras fork)